scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -472,20 +474,39 @@ static s32 pick_idle_cpu(struct task_struct *p, struct task_ctx *taskc,
 		goto out_put_cpumask;
 	}
 
+	// Greedy idle tries to grab any idle CPU in the LLC
+	if (greedy_idle && llcx->cpumask) {
+		cpu = scx_bpf_pick_idle_cpu(cast_mask(llcx->cpumask), 0);
+		if (cpu < nr_cpus && cpu >= 0) {
```

Comment:
Why are we checking for cpu < nr_cpus here? scx_bpf_pick_idle_cpu() should always return a cpu < nr_cpu_ids (or a negative value).

Reply:
Let me clean that up!